.github/workflows/test.yaml

```diff
@@ -170,7 +177,7 @@ jobs:
       # Related issue: https://github.com/Kong/kubernetes-testing-framework/issues/542
       run: |
         echo "TEST_KONG_IMAGE=kong/kong-gateway" >> $GITHUB_ENV
-        echo "TEST_KONG_TAG=3.1.1.3-alpine" >> $GITHUB_ENV
+        echo "TEST_KONG_TAG=3.2.1.0" >> $GITHUB_ENV
```

Comment:
I was about to comment that this won't use `alpine` based image but it seems there is no alpine image for 3.2. Just leaving a comment about it here for posterity.

Reply:
I asked to gateway team and they have deprecated the alpine based images. So for later usage we should switch to other images.